examples/demo/modules/blog/cms_blog.e

```diff
@@ -42,7 +48,7 @@ feature -- Access
 			Result := {CMS_BLOG_NODE_TYPE}.name
 		end
 
-feature -- Access: content
+feature -- Access: node
```

Comment:
I guess you did a copy/paste from node module.
just correcting typo  node vs content and later blog